.devcontainer/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM julia:1.7
```

Comment:
What is this file for?

Reply:
For github codespaces; Yggdrasil has it setup as well and it makes quick PR tests a breeze